Dockerfile

```diff
@@ -1,9 +1,9 @@
-ARG GOLANG="1.11.5"
-FROM golang:${GOLANG} as builder
+ARG GO_VERSION="1.11.5"
+FROM golang:${GO_VERSION} as builder
 
 ARG IMAGINARY_VERSION="dev"
 ARG LIBVIPS_VERSION="8.7.4"
-ARG GOLANG
+ARG GO_VERSION
```

Comment:
Superfluous, we already define it above

Reply:
We define it before the `FROM`, this statement brings `GO_VERSION` into this `FROM` scope.

```Dockerfile
ARG FOO
FROM ubuntu:latest
RUN echo "before ${FOO}."
ARG FOO
RUN echo "after ${FOO}."
```

```sh
$ docker build -t test --build-arg FOO=bar .
Step 1/5 : ARG FOO
Step 2/5 : FROM ubuntu:latest
 ---> 93fd78260bd1
Step 3/5 : RUN echo "before ${FOO}."
 ---> Running in 8eefaceba4a2
before .
Removing intermediate container 8eefaceba4a2
 ---> c73e8f12fd39
Step 4/5 : ARG FOO
 ---> Running in b7eb498de12a
Removing intermediate container b7eb498de12a
 ---> 4bfb6f66db1b
Step 5/5 : RUN echo "after ${FOO}."
 ---> Running in 8f3902108cf5
after bar.
Removing intermediate container 8f3902108cf5
 ---> 83fcfc578a6f
Successfully built 83fcfc578a6f
Successfully tagged test:latest

Reply:
Ah. Thanks for the `-vvv` response.